packages/trader/src/exchange-adapters/UniswapV3Trader.ts

```diff
@@ -0,0 +1,59 @@
+import winston from "winston";
+import Web3 from "web3";
+import BigNumber from "bignumber.js";
+
+const bn = require("bignumber.js"); // Big number that comes with web3 does not support square root.
+bn.config({ EXPONENTIAL_AT: 999999, DECIMAL_PLACES: 40 });
+
+const { encodePriceSqrt } = require("@uma/common");
+const ExchangeAdapterInterface = require("./ExchangeAdapterInterface");
+const { getTruffleContract } = require("@uma/core");
+
+class UniswapV3Trader implements InstanceType<typeof ExchangeAdapterInterface> {
+  readonly tradeDeadline: number;
+  readonly UniswapV2Broker: any;
```

Comment:
```suggestion
  readonly UniswapV3Broker: any;
```

Reply:
> Looks awesome! Super clean!
> 
> Some minor comments around typescript. Biggest one is preferring the `import X from Y;` (or `import { X } from Y;` syntax over `require`. I think the former will allow typescript to do a better job at resolving types.

@mrice32 Thank you for your feedback! I went back and refactored basically every file in the trader to follow the correct TS patterns. I also realized I had not updated the `hardhat.config.ts` as we'd discussed IRL which meant the unit tests were not correctly running in CI.

Please take a look again and let me know if you're happy :)